### PR TITLE
[IMP]10687 add module auth_signup_error_message.

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: auth_signup_error_message/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        if not qcontext.get('token') and not qcontext.get('signup_enabled'):
+            raise werkzeug.exceptions.NotFound()
+
+        if 'error' not in qcontext and request.httprequest.method == 'POST':
+            try:
+                self.do_signup(qcontext)
+                return super(AuthSignupHome, self).web_login(*args, **kw)
+            except (SignupError, AssertionError), e:
+                if e.message == self.password_error_message:
+                    qcontext['error'] = _(u'两次填写的密码不一致，请重新填写密码')
```
Comment (elicoidal) [inline auth_signup_error_message/controllers/main.py]: English please

File: auth_signup_error_message/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                else:
+                    qcontext['error'] = _(u'该用户名已经被注册，请重新填写用户名')
```
Comment (elicoidal) [inline auth_signup_error_message/controllers/main.py]: idem

**Review by elicoidal — CHANGES_REQUESTED** [review]
small details

**Comment by noahzaozao** [comment]
@Elico-Corp/odoo-reviewers Please review this PR again , fixed small details above.

**Comment by noahzaozao** [comment]
@elicoidal @seb-elico @victormartinelicocorp @xuan139

**Pull request merged**